core/installation/steps/select_modules.php

```diff
@@ -35,6 +35,10 @@
     $_SESSION['modules_selected'] = true;
     Redirect::to('?step=conversion');
 }
+
+if ($module !== 'Core') {
```

Comment:
This should be a function instead, which takes a $module parameter and returns the string, for two reasons:
- we shouldn't echo a variable which may not exist 
- $module will always be the same value at this point, since you did not place this logic in the for loop, no matter what module we are making html for, $module will be the same unchanged value (whatever was the last value of it in the for loop above)

Reply:
Core module should just not be listed at all is my point 😆

Reply:
The thing is it works, but unlisting it will be just kinda bad to hide from the user, no?